ec/src/lib.rs

```diff
@@ -32,328 +30,159 @@ use zeroize::Zeroize;
 pub mod models;
 pub use self::models::*;
 
-pub mod group;
+pub mod pairing;
+pub use self::pairing::*;
 
 pub mod msm;
 
 pub mod wnaf;
 
-pub trait PairingEngine: Sized + 'static + Copy + Debug + Sync + Send + Eq + PartialEq {
```

Comment:
The `PairingEngine` trait is renamed to `Pairing` and the implementation now lives in the `pairing` module.